pkgs/by-name/pr/proj-data/package.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, stdenv
+, callPackage
+, fetchFromGitHub
+
+  # proj-data grid directories to install (ex.: "nz_linz").
+  # By default, no grids are installed due to the large size (size of all proj-
+  # data grids is nearly 1GB and will grow over the time).
+, gridPackages ? [ ]
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "proj-data";
+  version = "1.16.0";
+
+  src = fetchFromGitHub {
+    owner = "OSGeo";
+    repo = "PROJ-data";
+    rev = finalAttrs.version;
+    hash = "sha256-/EgDeWy2+KdcI4DLsRfWi5OWcGwO3AieEJQ5Zh+wdYE=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    shopt -s extglob
+
+    mkdir -p $out
+    cp README.DATA $out/README-PROJ-DATA.md
+
+    for grid in ${builtins.toString gridPackages}; do
+      if [ ! -d $grid ]; then
+        echo "ERROR: Grid ($grid) does not exist."
+        exit 1
+      fi
+
+      cp $grid/!(*.sh|*.py) $out/
+    done
+
+    shopt -u extglob
+    runHook postInstall
+  '';
+
+  passthru.tests =
+    let
+      # build custom package containing `nz_linz` grids
+      projDataWithGrid = finalAttrs.finalPackage.overrideAttrs (_: {
+        postInstall = "cp nz_linz/* $out/";
```

Comment:
Shouldn't this override `gridPackages` instead?

Reply:
Yes, I would like to, but `finalPackage` doesn't support `override` (it supports only `overrideAttrs`). Any suggestions are welcome.

Reply:
I don't know Nix well enough to suggest anything. Feel free to ignore this, unless someone else has a suggestion.